src/Logging.W3C/src/W3CLoggerOptions.cs

```diff
@@ -0,0 +1,72 @@
+using System;
+
+namespace Microsoft.Extensions.Logging.W3C
+{
+    /// <summary>
+    /// Options for the <see cref="W3CLogger"/>.
+    /// </summary>
+    public class W3CLoggerOptions
+    {
+        private int? _fileSizeLimit = 10 * 1024 * 1024;
+        private string _fileName = "serverlog-";
+        private string _logDirectory = "C:\\code\\scratch\\W3CLogs";
```

Comment:
TODO - where should these logs be written to? This is just the value I used for testing

Reply:
Maybe `/logs` relative to current directory - ANCM does this. Is it content_root or www_root?

Reply:
content_root